internal/start/start.go

```diff
@@ -770,10 +770,12 @@ EOF
 				Image: utils.PgbouncerImage,
 				Env: []string{
 					"POSTGRESQL_HOST=" + dbConfig.Host,
-					"POSTGRESQL_NAME=" + dbConfig.Database,
-					"POSTGRESQL_USER=" + dbConfig.User,
 					fmt.Sprintf("POSTGRESQL_PORT=%d", dbConfig.Port),
+					"POSTGRESQL_USERNAME=pgbouncer",
```

Comment:
Not `dbConfig.User`?

Reply:
Yup, this is for pgbouncer to connect and fetch the actual user password from postgres.